source/Makefile

```diff
@@ -30,3 +30,11 @@ clean-examples:
 	rm -f $(EXAMPLES)
 
 examples: $(EXAMPLES)
+
+.PHONY: check
```

Comment:
Aren't the "clean" targets above also "phony"? Can you move the .PHONE line up to the top?